app/modules/import-export/processes/revert.php

```diff
@@ -25,8 +25,8 @@ class Revert {
 	 * @throws \Exception
 	 */
 	public function __construct() {
-		$this->import_sessions = $this->get_import_sessions();
-		$this->revert_sessions = $this->get_revert_sessions();
+		$this->import_sessions = self::get_import_sessions();
+		$this->revert_sessions = self::get_revert_sessions();
```

Comment:
If there is no specific reason to use 'self' here I prefer to use 'static' to allow easier extension to this class in the future

Reply:
Actually the opposite, if there is no reason to use static you should use self, just like if there is no reason to use "protected" function you should use "private" unless you think someone should use it.
No one should change those functions they only get the data from options, this will probably last forever.